### PR TITLE
Fixes #187 Invalid Date

### LINKED ISSUE #187: Liquid Exception: Invalid Date
Using the latest master branch (f048342), I'm getting this error related to `LastModifiedAt`:

```
Liquid Exception: Invalid Date: '#<Jekyll::LastModifiedAt::Determinator:0x000000034cc770 @site_source="/home/kyle/website/aav7fl.github.io", @page_path="/home/kyle/website/aav7fl.github.io/_posts/2013-04-13-somewhere-to-start.md", @opts={}>' is not a valid datetime. in /_layouts/default.html
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    Invalid Date: '#<Jekyll::LastModifiedAt::Determinator:0x000000034cc770 @site_source="/home/kyle/website/aav7fl.github.io", @page_path="/home/kyle/website/aav7fl.github.io/_posts/2013-04-13-somewhere-to-start.md", @opts={}>' is not a valid datetime.
[17:44:02] 'build' errored after 790 ms
```

Still investigating, but I didn't want a new gem to get pushed out before we figured it out.

Edit 1: It looks to be coming from https://github.com/gjtorikian/jekyll-last-modified-at/blob/master/lib/jekyll-last-modified-at/determinator.rb and how the date is passed into the new template.

`date_modified` in `drop.rb` looks to be the source of the issue.

### PR DESCRIPTION
**Pull request by aav7fl**
Implements suggested change from @benbalter (and works). Reference issue: #187

### CODE REVIEW

**Comment by aav7fl** [comment]
I'm working on the other failing tests right now. Just a heads up. I have one of them, but still working on the author. I think I just figured out what is happening, now I just need to figure out why.

File: lib/jekyll-seo-tag/drop.rb [hunk 1]
```diff
@@ -98,7 +98,7 @@ def date_modified
           date = if page["seo"] && page["seo"]["date_modified"]
                    page["seo"]["date_modified"]
                  else
-                   page["last_modified_at"] || page["date"]
+                   page["last_modified_at"].to_liquid || page["date"]
```
Comment (benbalter) [inline lib/jekyll-seo-tag/drop.rb]: Actually, this needs to be 

```ruby
elsif page["last_modified_at"]
  page["last_modified_at"].to_liquid
else
  page["date"]
```

to handle the case where `last_modified_at` is `nil` (where calling `to_liquid` on `nil` would throw an error)

**Comment by aav7fl** [comment]
Everything should be all *up to date* (pun intended) here.

**Comment by DirtyF** [comment]
@jekyllbot: merge +minor

**Comment by benbalter** [comment]
Thanks again @aav7fl!

**Pull request merged**